util/config_file.h

```diff
@@ -362,6 +362,11 @@ struct config_file {
 	int serve_expired_ttl;
 	/** reset serve expired TTL after failed update attempt */
 	int serve_expired_ttl_reset;
+	/** TTL for the serve expired replies */
+	int serve_expired_reply_ttl;
+	/** serve expired entries only after trying to update the entries and this
+	 *  timeout is reached */
```

Comment:
add `in milliseconds`.  Also the serve_expired_reply_ttl and serve_expired_client_timeout do not have a recommended value shown (commented out) in doc/example.conf.in.

Reply:
Fixed in https://github.com/NLnetLabs/unbound/pull/159/commits/479b4c69e75ec22460263dcee159e9e749423273